src/gov/nist/javax/sip/stack/UDPMessageChannel.java

```diff
@@ -318,6 +318,9 @@ public void run() {
 
                 logger.logError(
                         "Error while processing incoming UDP packet" + Arrays.toString(packet.getData()), e);
+            } catch (StackOverflowError e) {
+                logger.logError("StackOverflowError while processing incoming UDP packet" + Arrays.toString(packet.getData()));
```

Comment:
Why not add the exception at the end of the log line like line 320? and delete the double log

Reply:
The reason is that UDPMessageChannel is not using a typical `java.util.logging.Logger` but a `gov.nist.core.StackLogger`. This logger API is not as flexible. 

Here:

![image](https://github.com/jpgiroux/jsip/assets/8933076/662e3ea2-ae96-48ec-a23c-07b444853b99)

Reply:
I have refactor the `logError(String, Exception): void` for `logError(String, Throwable): void` so now the catch clause is cleaner.